### PR TITLE
[MM-46251] Change activeUsers type: string -> string[]

### PR DESCRIPTION
**Pull request by plant99**
#### Summary

`activeUsers` field in TopBoardInsights contain a list of user ids but is of type string. This PR changes that type to []string.
#### Ticket Link

https://mattermost.atlassian.net/browse/MM-46251
#### Related Pull Requests

- Has server changes [(link here)](https://github.com/mattermost/focalboard/pull/3909)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by BenCookie95 — APPROVED** [review]

**Review by cyrilzhang-mm — APPROVED** [review]

**Comment by plant99** [comment]
Waiting for Martin's review on https://github.com/mattermost/focalboard/pull/3909 before requesting QA review in both PRs.

**Review by furqanmlk — APPROVED** [review]
working fine

**Review by furqanmlk — APPROVED** [review]

**Review by BenCookie95 — APPROVED** [review]

File: packages/types/src/insights.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ export type TopBoard = {
     icon: string;
     title: string;
     activityCount: number;
-    activeUsers: string;
+    activeUsers: string[];
```
Comment (M-ZubairAhmed) [inline packages/types/src/insights.ts:100]: absolutely nothing to change here, It is helpful to strict type this more by actually having this tied to the `UserProfile`'s id.

Comment (plant99) [reply packages/types/src/insights.ts]: Done.

**Pull request merged**